### PR TITLE
fixes #133

### LINKED ISSUE #133: Add method telling if policy is enforceable
It is useful to know if resulting policy is "enforceable" or "report" policy.
For example, `form-action *; ` is a redundant policy so it does not contain any enforceable directives.

### PR DESCRIPTION
**Pull request by shekyan**
- add policy method to tell if it is effective policy
- tests
- remove unused imports

### CODE REVIEW

**Comment by michaelficarra** [comment]
I think you're conflating two different concepts. A policy's disposition appears to be wholly determined by whether it was delivered via a `Content-Security-Policy` or `Content-Security-Policy-Report-Only` directive. This property seems valuable, but we should not name it anything related to "enforce" or "disposition".

**Comment by shekyan** [comment]
Renamed to `isEffective`

**Comment by michaelficarra** [comment]
How about `hasSomeEffect`?

**Comment by shekyan** [comment]
Yeah I like it.

**Comment by shekyan** [comment]
Changed to `hasSomeEffect`

File: src/test/java/com/shapesecurity/salvation/PolicyQueryingTest.java [hunk 1]
```diff
@@ -426,4 +426,12 @@
         assertFalse(p.allowsScriptFromSource(new GUID("data:")));
     }
 
+    @Test public void testIsEnforceable() {
```
Comment (michaelficarra) [inline src/test/java/com/shapesecurity/salvation/PolicyQueryingTest.java]: This name needs to change.

**Comment by michaelficarra** [comment]
Post-merge tests are failing. Try rebasing on master.

**Comment by shekyan** [comment]
Good to go.

File: src/test/java/com/shapesecurity/salvation/PolicyQueryingTest.java [hunk 2]
```diff
@@ -426,4 +426,12 @@
         assertFalse(p.allowsScriptFromSource(new GUID("data:")));
     }
 
+    @Test public void testHasSomeEffect() {
+        Policy p = Parser.parse("script-src *", "http://example.com");
```
Comment (michaelficarra) [inline src/test/java/com/shapesecurity/salvation/PolicyQueryingTest.java]: I would prefer just the empty policy here. Remember, we're changing this. `script-src *` doesn't allow inline scripts.

Comment (shekyan) [inline src/test/java/com/shapesecurity/salvation/PolicyQueryingTest.java]: Agree. Done

**Pull request merged**